proxy/http/HttpDebugNames.cc

```diff
@@ -232,6 +211,76 @@ HttpDebugNames::get_event_name(int event)
     return "HTTP_SCH_UPDATE_EVENT_ERROR";
   case HTTP_SCH_UPDATE_EVENT_NO_ACTION:
     return "HTTP_SCH_UPDATE_EVENT_NO_ACTION";
+
+#define X(SUFFIX) N1(TS_EVENT_##SUFFIX)
```

Comment:
I thought we said @SolidWallOfCode will have a stroke when we use #define? :-)

Reply:
It's clear you all fear my templates more than Walt's macro-oni. Do note that Walt, not being _criminally_ insane, cleans up his `#define` names so they don't cause problems later.

Reply:
So is this the consensus then, I should not use any macros out of respect for everyone's religious beliefs?

Reply:
I don't think Leif counts as a "consensus" all by himself.

Reply:
So only zwoop and maskit fear for my immortal soul for using macros not you?